R/class-Megastudy.R

```diff
@@ -85,13 +92,16 @@ check_megastudy <- function(object) {
 #' that data.
 #' 
 #' @slot data A data.table
+#' @slot ancestorIdColumns A character vector of column names representing parent entities of the recordIdColumn.
 #' @slot studySpecificVocabularies veupathUtils::StudySpecificVocabulariesByVariableList
+#' @slot collectionIds A character vector of collection ids we expect. If none provided, they will be inferred from those present.
```

Comment:
I'm a bit confused. At [this line](https://github.com/VEuPathDB/veupathUtils/pull/44/files#diff-bfa560be7c1096782e219687265a49a74ca7dcecf7cf900a96ef999af7b08295R263) we are pulling variable names out of `collectionsDT` - but here it just says it's a vector of collection IDs. Is it actually a table of variable values?

Reply:
Yea I changed my mind about how things would work and forgot to update the docs. I'll fix it.